aws-ssm-patchbaseline/.gitignore

```diff
@@ -12,9 +12,15 @@ out.java
 out/
 .settings
 .project
-
+*.whl
 # auto-generated files
 target/
+*_internal.json
```

Comment:
do we still have internal files?

Reply:
Internal json file are removed and removed the entry in gitignore